src/components/step/StepImage/StepImage.css

```diff
@@ -0,0 +1,10 @@
+.mdhui-step-container .mdhui-step-image {
+    text-align: center;
+}
+.mdhui-step-container .mdhui-step-image img {
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    padding-top: 40px;
```

Comment:
For the existing step images we don't set padding-bottom. Is this needed to account for styling differences in the other stop components?

Reply:
Yes; I'd have to look back and do the comparison - but on our normal InstructionStep there's some padding of the container that contains the title/text/detail portion, which is absent in the Step components here.  Seemed slightly easier to just handle it here, rather than have to re-arrange the other CSS and potentially affect the existing YouTube step.